btrack/btypes.py

```diff
@@ -454,55 +446,53 @@ def softmax(self) -> list:
 
     @property
     def is_root(self) -> bool:
-        return (
-            self.parent == 0 or self.parent is None or self.parent == self.ID
-        )
+        return self.parent == 0 or self.parent is None or self.parent == self.ID
 
     @property
     def is_leaf(self) -> bool:
         return not self.children
 
     @property
-    def kalman(self) -> np.ndarray:
+    def kalman(self) -> npt.NDArray:
         return self._kalman
 
     @kalman.setter
-    def kalman(self, data: np.ndarray) -> None:
+    def kalman(self, data: npt.NDArray) -> None:
         assert isinstance(data, np.ndarray)
         self._kalman = data
 
-    def mu(self, index: int) -> np.ndarray:
+    def mu(self, index: int) -> npt.NDArray:
         """Return the Kalman filter mu. Note that we are only returning the mu
         for the positions (e.g. 3x1)."""
         return self.kalman[index, 1:4].reshape(3, 1)
 
-    def covar(self, index: int) -> np.ndarray:
+    def covar(self, index: int) -> npt.NDArray:
         """Return the Kalman filter covariance matrix. Note that we are
         only returning the covariance matrix for the positions (e.g. 3x3)."""
         return self.kalman[index, 4:13].reshape(3, 3)
 
-    def predicted(self, index: int) -> np.ndarray:
+    def predicted(self, index: int) -> npt.NDArray:
```

Comment:
At some point would be good to type what these are, i.e. `npt.NDArray[np.float_]`, but let's leave this for now.